java8homework/src/main/java/RandomGenerator.java

```diff
@@ -0,0 +1,22 @@
+import java.security.SecureRandom;
+import java.util.Objects;
+
+public class RandomGenerator {
+    private static final String ALPHABET = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+    private static final SecureRandom RANDOM = new SecureRandom();
+
+    /**
+     * Generates random string of given length from Base65 alphabet (numbers, lowercase letters, uppercase letters).
+     *
+     * @param count length
+     * @return random string of given length
+     */
+
+    public static String generator(int count) {
+        StringBuilder sb = new StringBuilder();
+        for (int i = 0; i < count; ++i) {
+            sb.append(ALPHABET.charAt(RANDOM.nextInt(ALPHABET.length())));
+        }
+        return sb.toString();
+    }
+}

```

Comment:
-- on the deadline (very very late), but besides that, the code looks great (with minor observations) but I can see that you've worked hard for this implementation, good job!